io.github.piotrek_k._2dTaskBoard.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.github.piotrek_k._2dTaskBoard</id>
+
+    <name>2dTaskBoard</name>
+    <summary>Task management productivity app with markdown support</summary>
+
+    <metadata_license>CC-BY-4.0</metadata_license>
+    <project_license>GPL-3.0-only</project_license>
+
+    <url type="homepage">https://piotrek-k.pl/2dTaskBoard</url>
+    <url type="bugtracker">https://github.com/piotrek-k/2dTaskBoard/issues</url>
+
+    <supports>
+        <control>pointing</control>
+        <control>keyboard</control>
+        <control>touch</control>
+    </supports>
+
+    <description>
+        <p>With 2dTaskBoard you can track the progress of your daily tasks. Key features:</p>
+        <ul>
+            <li>2D Board - move tasks between rows and columns.</li>
+            <li>File system based - all your data are files which can be then synced using cloud providers or git.</li>
+            <li>Markdown support - rich text formatting.</li>
+            <li>Mermaid support - flowcharts rendering.</li>
+            <li>Attachments - attach files of any kind to your tasks.</li>
+            <li>Obsidian compatibility - open tasks in Obsidian and link to existing notes.</li>
+            <li>Full keyboard support - every part of this app is accessible just by pressing keyboard shortcuts</li>
+        </ul>
+    </description>
+
+    <launchable type="desktop-id">io.github.piotrek_k._2dTaskBoard.desktop</launchable>
+
+    <releases>
+        <release version="0.4.1" date="2025-03-15">
+            <description></description>
+        </release>
+    </releases>
+
+    <developer id="pl.piotrek-k">
+        <name>Piotr Kozerski</name>
+    </developer>
+
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/piotrek-k/flathub/refs/heads/my-app-submission/screenshots/board_view.png</image>
```

Comment:
screenshot URL should never reference the tip of a branch. It's not immutable.

Reply:
Now it's referring to commit, let me know if this is what you meant